arch/Kconfig

```diff
@@ -222,6 +222,57 @@ config ARCH_TOOLCHAIN_GNU
 	bool
 	default n
 
+config ARCH_TOOLCHAIN_CLANG
```

Comment:
let's add a new patch to support clang on armv6-m too.

Reply:
Done